lib/route.coffee

```diff
@@ -57,26 +74,15 @@ class share.Route
           responseData = null
           try
             responseData = self._callEndpoint endpointContext, endpoint
+            if responseData instanceof Promise
```

Comment:
I think you can use `Promise.resolve` to handle with the same syntax the case where the result is a `Promise` and when it isn't. But of course it works the way you did it ;)

```coffee
Promise.resolve(responseData).then (result) ->
  respond result, {responseInitiated, res}
```
I don't really understand what the `try catch` block is for though, so maybe I'm wrong.

Reply:
is this package about to publish a new version?!!